apps/crm/src/lib/metadata-manager.ts

```diff
@@ -88,7 +99,8 @@ export async function mutateMetadata(
 }
 
 export const createSettingsManager = (
-  client: Pick<Client, "query" | "mutation">
+  client: Pick<Client, "query" | "mutation">,
+  appId: string
```

Comment:
@krzysztofwolski @peelar in first metadata manager implementation we didnt have app ID in auth data, so `set` was fetching app ID before putting metadata. It is unnecessary now. We can add appID to settings manager constructor